app/js/lib/form_pages.js

```diff
@@ -0,0 +1,89 @@
+var Promise = require( 'promise' ),
+	jQuery = require( 'jquery' ),
+
+	FormPage = {
+		show: function () {
+			jQuery( this.sections ).show();
+		},
+
+		hide: function () {
+			jQuery( this.sections ).hide();
+		},
+		validate: function () {
+			'use strict';
```

Comment:
There was a point where one of the code quality tools did complain and I found some some explanation that putting it at the beginning of the file is not good style because when the files are concatenated the there are all these "use strict" statements in the middle of the code. However, now that I checked, the tools don't complain any more. and Browserify does something more intelligent than just concatenating. I'll rework this.